resources/views/app.blade.php

```diff
@@ -30,10 +30,12 @@
 				<img src="/assets/img/laravel-auditing-logo.png" alt="Laravel Auditing logo" style="margin-left: 15px;margin-right: 0;height: 40px;top: 14px;">
 				Laravel Auditing
 			</a>
-			
-			<div class="responsive-sidebar-nav">
-				<a href="#" class="toggle-slide menu-link btn">&#9776;</a>
-			</div>
+
+			@if (!Request::is('/'))
+				<div class="responsive-sidebar-nav">
```

Comment:
I think you should use the class hidden-xs

Reply:
Yes, possibly the better solution, let me amend.

Reply:
Actually, no, why would you want that button rendered on / at all? 

We'll need to do some ugliness like that 
```
<div class="responsive-sidebar-nav {{ Request::is('/') ? 'hidden-xs hidden-sm' : '' }}">
    <a href="#" class="toggle-slide menu-link btn">&#9776;</a>
</div>
```

Reply:
Sorry, you're right. It was my mistake.